addons/hr_expense/models/hr_expense.py

```diff
@@ -516,14 +516,14 @@ def _check_payment_mode(self):
         for sheet in self:
             expense_lines = sheet.mapped('expense_line_ids')
             if expense_lines and any(expense.payment_mode != expense_lines[0].payment_mode for expense in expense_lines):
-                raise ValidationError(_("Expenses must have been paid by the same entity (Company or employee)"))
+                raise ValidationError(_("Expenses must be paid by the same entity (Company or employee)"))
```

Comment:
I forgot the . at the end of the sentence : "Expenses must be paid by the same entity (Company or employee)."